src/components/organisms/Socials.tsx

```diff
@@ -55,6 +55,11 @@ const Social = styled(Link)`
     width: 40px;
   }
 
+  svg,
+  img {
+    filter: grayscale(100) brightness(100);
```

Comment:
I noticed that the social logos weren't being rendered anymore, and I tracked the issue down to this CSS code.

The value `100` is incorrect — it should be either `1` or `100%`. See the [Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/CSS/filter-function/grayscale).

<img width="250" alt="SCR-20241212-lgvd" src="https://github.com/user-attachments/assets/618cc67b-2f57-4aee-8a7c-7f3e728f597b" />

Reply:
Good catch. They were working fine in the dark mode so I failed to notice this. Fixed in https://github.com/sablier-labs/docs/pull/186/commits/3947ee2355f5575c6149024fd6ad70d60049d0a2.

Reply:
There was still another issue with the GitHub icon, I pushed a commit to fix it.